### PR TITLE
Reduce traceInfo maxBatchesPerWrite

### PR DESCRIPTION
**Pull request by zalegrala**
Here we reduce the number of batches that could get sent to Tempo by default.
Without this change, we increase the potential of sending too large of traces
for the defaults of the Tempo service.

**Checklist**
- [x] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: pkg/util/trace_info.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ var (
 	// that we get the expected number of batches on a trace.  A value larger
 	// than 25 here results in vulture writing traces that exceed the maximum
```
Comment (mapno) [inline pkg/util/trace_info.go:20]:
```suggestion
	// than 10 here results in vulture writing traces that exceed the maximum
```

**Review by mapno — COMMENTED** [review]
Can this be configured or calculated somehow? Instead of a hardcoded value.

**Comment by zalegrala** [comment]
One way I was thinking about retrieving this value would be to query the API of Tempo to know what the configured value is and then determine something appropriate here.

**Comment by joe-elliott** [comment]
> One way I was thinking about retrieving this value would be to query the API of Tempo to know what the configured value is and then determine something appropriate here.

Feels too complex to me. Fine with just reducing the default.

**Review by joe-elliott — APPROVED** [review]

**Pull request merged**